app/models/miq_report/generator.rb

```diff
@@ -88,6 +88,34 @@ def get_include
     include_as_hash(include.presence || invent_report_includes)
   end
 
+  def polymorphic_includes
+    @polymorphic_includes ||= begin
+      polymorphic_rels = []
+      top_level_rels   = []
+      top_level_rels  += Array(get_include.try(:keys))
+      top_level_rels  += Array(get_include_for_find.try(:keys))
+
+      top_level_rels.uniq.each do |assoc|
```

Comment:
feels like an `each_with_object([]) do |assoc, polymorphic_rels` would read better

(also delete 2 lines and a white space) ✂️